snakemake/shell.py

```diff
@@ -101,7 +101,9 @@ def __new__(
         shadow_dir = context.get("shadow_dir", None)
 
         cmd = "{} {} {}".format(
-            cls._process_prefix, cmd.strip(), cls._process_suffix
+            format(cls._process_prefix, stepout=2),
```

Comment:
I should probably add the args and kwargs here?
```
format(cls._process_prefix, *args, stepout=2, **kwargs),
```

Reply:
Yes, please define a lambda above, which you then use for cmd, prefix and suffix.

Reply:
53472922a5664ff8748843ff52c5a8f6ff841f09